lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class NetworkTokenizationCreditCard < CreditCard
+      # A +NetworkTokenizationCreditCard+ object represents a tokenized credit card 
+      # using the EMV Network Tokenization specification, http://www.emvco.com/specifications.aspx?id=263.
+      #
+      # It includes all fields of the +CreditCard+ class with additional fields for 
+      # verification data that must be given to gateways through existing fields (3DS / EMV).
+      #
+      # The only tested usage of this at the moment is with an Apple Pay decrypted PKPaymentToken, 
+      # https://developer.apple.com/library/ios/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html 
```

Comment:
Can you add some clarification commentary here and in `PaymentToken`? 
That is -- if I'm correct -- some language to describe that: 
- `NetworkTokenizationCreditCard` is exactly like a credit card but with EMV/3DS standard payment network tokenization data
- `PaymentToken` is meant for modeling proprietary payment token structures (like `stripe` and `apple_pay`)